### PR TITLE
chore: refactor new room sound

### PR DESCRIPTION
**Pull request by ggazzo**
found during this issue:
[rocketchat.atlassian.net/browse/CORE-811](https://rocketchat.atlassian.net/browse/CORE-811)

### CODE REVIEW

**Review by aleksandernsilva — CHANGES_REQUESTED** [review]
Considering the current PR title, changing it to `refactor: new room sound` might be a better fit.

File: apps/meteor/client/hooks/useOmnichannelContinuousSoundNotification.ts [hunk 1]
```diff
@@ -16,6 +16,8 @@ export const useContinuousSoundNotification = () => {
 
 	const continuousCustomSoundId = newRoomNotification && `${newRoomNotification}-continuous`;
 
+	const hasUnreadRoom = userSubscriptions.length > 0 && queue.length > 0;
```
[the remark this answers is not in the document]
Comment (ggazzo) [reply apps/meteor/client/hooks/useOmnichannelContinuousSoundNotification.ts:19]: yep sorry

Comment (ggazzo) [inline apps/meteor/client/hooks/useOmnichannelContinuousSoundNotification.ts:19]:
```suggestion
	const hasUnreadRoom = userSubscriptions.length > 0 || queue.length > 0;
```

File: apps/meteor/client/hooks/useNotifyUser.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ export const useNotifyUser = () => {
 		}
 
 		if ((!router.getRouteParameters().name || router.getRouteParameters().name !== sub.name) && !sub.ls && sub.alert === true) {
-			KonchatNotification.newRoom(sub.rid);
+			KonchatNotification.newRoom();
```
[the remark this answers is not in the document]
Comment (ggazzo) [reply apps/meteor/client/hooks/useNotifyUser.ts:25]: not anymore the rid was used to be added to Session, but I removed that `Session`

**Review by aleksandernsilva — APPROVED** [review]

**Pull request merged**